service/handlers.go

```diff
@@ -27,6 +30,39 @@ import (
 	"go.uber.org/zap"
 )
 
+func handleQuery(c *Core, w *ResponseWriter, r *Request) {
+	var req api.QueryRequest
+	if !r.Unmarshal(w, &req) {
+		return
+	}
+	zquery, err := query.ParseSources(req.Query, req.Includes)
```

Comment:
This reads from the file system, so it needs to happen in the client.

Reply:
Cool